### ISSUE TITLE
Class static used to pass info between stream steps

### ISSUE THREAD

**Issue by jukkakarvanen** [kaiwaehner/kafka-streams-machine-learning-examples#12]
Kafka_Streams_TensorFlow_Image_Recognition_Example use class static variables to pass information between foreach and mapValues steps.

So I expect rather interesting problems if executed with multiple threads.

I have fix for this, but it is on top of changes related to #11

**Comment by jukkakarvanen** [comment]
Similar problems noticed also in other classes.

**Comment by kaiwaehner** [comment]
Yeah, to be honest, my first implementation was "quick and dirty". Thank you so much for these improvements and pull requests.

**Comment by kaiwaehner** [comment]
I merged your fix. Thank you. 

I will keep the ticket open as reminder for the other classes. I will refactor the other classes, too.

**Comment by hamzakhalil798** [comment]
@jukkakarvanen 
Hey! im a newbie at this... im trying to run this repo using mvn clean package. Do I have to use kafka as well? or just use mvn clean package and everything would run?
kindly help me out on this.
